app/services/export_activity_to_csv.rb

```diff
@@ -56,4 +56,17 @@ def headers
       "Link to activity in RODA",
     ].concat(next_four_quarters).to_csv
   end
+
+  def next_four_quarter_forecasts
+    quarter_date_ranges = quarters_to_date_ranges(next_four_quarters)
+    quarter_date_ranges.map { |range| activity_presenter.forecasted_total_for_date_range(range: range) }
+  end
+
+  private def activity_presenter
+    @activity_presenter ||= ActivityPresenter.new(activity)
+  end
+
+  private def next_four_quarters
```

Comment:
Could this just be included in `next_four_quarter_forecasts` ..?

Reply:
One is the headers and one is the results but yes maybe we can have a hash rather than an array

Reply:
On second thoughts probably not really - the headers belong to the *report* but the results belong to the *activity* so I don't know if returning both of them in the same object is going to make sense?